.travis.yml

```diff
@@ -1,20 +1,16 @@
 os:
   - linux
 language: node_js
+dist: bionic
+services:
+  - xvfb
 node_js:
-  - '10'
-  - '8'
-env: DISPLAY=:99.0
-install: true
-addons:
-  apt:
-    packages:
-    - metacity
```

Comment:
Why can't we use the `apt` Travis config?

Reply:
Not sure why, maybe because it's not listed in https://github.com/travis-ci/apt-source-safelist

Reply:
check https://blog.travis-ci.com/2018-11-08-xenial-release 
and https://docs.travis-ci.com/user/reference/xenial/#third-party-apt-repositories-removed

Reply:
👍